src/aklite_client_ext.cc

```diff
@@ -26,25 +27,25 @@
 // Returns the target that should be installed, if any.
 // It might be an updated version, a rollback target, or even the currently installed target, in case we need to sync
 // apps
-GetTargetToInstallResult AkliteClientExt::GetTargetToInstall(const LocalUpdateSource* local_update_source, int version,
+GetTargetToInstallResult AkliteClientExt::GetTargetToInstall(const CheckInResult& check_in_result, int version,
```

Comment:
I would call this parameter as `checkin_result` or `checkin_res` since this is a result of the `CheckIn` call, we kind of use "check" and "in" together.